pybamm/models/submodels/current_collector/homogeneous_current_collector.py

```diff
@@ -23,13 +23,21 @@ def __init__(self, param):
 
     def get_fundamental_variables(self):
 
-        whole_cell = ["negative electrode", "separator", "positive electrode"]
-        i_cc = pybamm.FullBroadcast(pybamm.Scalar(0), whole_cell, "current collector")
-
+        # TODO: grad not implemented for 2D yet
+        i_cc = pybamm.Scalar(0)
         i_boundary_cc = pybamm.PrimaryBroadcast(
```

Comment:
for 1D do we expect the `i_boundary_cc` to be `current_with_time` or `current_with_time/cc_area` ? Not for this issue, but I'm not sure if it would be more user friendly to provide cell capacity in the csv files and then set the typical current to be the 1C discharge current and C-rate to be 1 by default (obviously the user can adjust these later)

Reply:
In 1D I think it should be (dimensionless) `current_with_time`, which is the same as dimensionless `current_density_with_time` but not the same as `current_with_time/cc_area`.
Don't completely understand the point about 1C discharge currents?

Reply:
ah ok. for the 1C thing I was just thinking that for users it might seem strange to provide the typical current in a parameter file, and may be more natural to give the cell capacity as the parameter, which then sets the default typical current (to the 1C discharge current). maybe this is a bit of a non-issue though

Reply:
Ok - I guess ideal scenario is you can do either and it updates the other. Isn't most battery data current vs voltage rather than Crate vs voltage though?

Reply:
Even if you are reading in current vs voltage data you still only set one typical current used to non-dimensionlise. As you say, setting either would be best. In any case, I think the default current shouldn't be set to 1A and should instead be set based on the cell capacity (if provided). Then if you are going to be loading in current vs voltage data you might want to change this to a typical current based on your data and then the C-rate changes accordingly (I feel like I'm not explaining this very well haha)